judge/testcase_run.sh

```diff
@@ -291,13 +291,11 @@ resourceinfo="\
 runtime: ${program_cputime}s cpu, ${program_walltime}s wall
 memory used: ${memory_bytes} bytes"
 
-if [ $COMBINED_RUN_COMPARE -eq 1 ] && grep '^exitcode: 43' compare.meta > /dev/null 2>&1 ; then
+if [ $COMBINED_RUN_COMPARE -eq 1 ] && grep '^validator-exited-first: true' compare.meta > /dev/null 2>&1 && grep '^exitcode: 43' compare.meta > /dev/null 2>&1 ; then
```

Comment:
For the others: I added this. @edomora97 his code always created the meta file and since we had a FIXME here already anyway, I decided to leave it like that and check it. We might want to improve on it even more in the future, but this is already better than before.

Reply:
Sounds good